walkthroughs/tls-with-acm/README.md

```diff
@@ -10,19 +10,7 @@ With ACM, you can host some or all of your Public Key Infrastructure (PKI) in AW
 
 Let's jump into a brief example of App Mesh TLS in action.
 
-## Step 1: Download the App Mesh Preview CLI
-
-You will need the latest version of the App Mesh Preview CLI for this walkthrough. You can download and use the latest version using the command below.
-
-```bash
-aws configure add-model \
-        --service-name appmesh-preview \
-        --service-model https://raw.githubusercontent.com/aws/aws-app-mesh-roadmap/master/appmesh-preview/service-model.json
-```
-
-Additionally, this walkthrough makes use of the unix command line utility `jq`. If you don't already have it, you can install it from [here](https://stedolan.github.io/jq/).
-
-## Step 2: Create Color App Infrastructure
+## Step 1: Create Color App Infrastructure
 
 We'll start by setting up the basic infrastructure for our services. All commands will be provided as if run from the same directory as this README.
 
```

Comment:
Can't comment on the line. ssh complains about the default permissions set on the pem file. Can add `chmod go-r ~/.ssh/color-app.pem` to fix it.

Reply:
Done